ecocodejava.properties

```diff
@@ -15,7 +14,13 @@ defaults.mavenArtifactId=ecocode-java-plugin
 1.2.1.changelogUrl=https://github.com/green-code-initiative/ecoCode/releases/tag/1.2.1
 
 1.3.0.description=Update rules tags�; upgrade librairies to SonarQube 10.0.0�; clean-up plugins and dependencies
-1.3.0.sqVersions=[9.9,LATEST]
+1.3.0.sqVersions=[9.9,10.1]
 1.3.0.date=2023-07-04
 1.3.0.downloadUrl=https://github.com/green-code-initiative/ecoCode/releases/download/1.3.0/ecocode-java-plugin-1.3.0.jar
 1.3.0.changelogUrl=https://github.com/green-code-initiative/ecoCode/releases/tag/1.3.0
+
+1.4.2.description=A lot of corrections and improvements (please check CHANGELOG)
+1.4.2.sqVersions=[9.4,9.8,9.9,10.0,10.1]
```

Comment:
Please see the README for this repo to understand what valid values are for this.
Also, the current LTS is 9.9. That's the floor for listing compatibility.

Reply:
modification done